bazel/repository_locations.bzl

```diff
@@ -651,12 +651,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Rules for using foreign build systems in Bazel",
         project_desc = "Rules for using foreign build systems in Bazel",
         project_url = "https://github.com/bazelbuild/rules_foreign_cc",
-        version = "d54c78ab86b40770ee19f0949db9d74a831ab9f0",
-        sha256 = "e7446144277c9578141821fc91c55a61df7ae01bda890902f7286f5fd2f6ae46",
+        version = "da8952e27c46659866b455de58f5eb3a86185507",
+        sha256 = "5d1e34fc0f4ab4af0e595ee4d9a03f4127701d95c1d521296bf892b8aff903a2",
         strip_prefix = "rules_foreign_cc-{version}",
         urls = ["https://github.com/bazelbuild/rules_foreign_cc/archive/{version}.tar.gz"],
-        release_date = "2020-10-26",
-        use_category = ["build"],
+        release_date = "2021-08-21",
+        use_category = ["build", "dataplane_core", "controlplane"],
```

Comment:
Is this actually required by the dependency checker? LGTM otherwise.

Reply:
yea, we probably need to investigate out of band, it seems the toolchains changes made in this repo make it so there's a connection from these where I'm surprised to see it. This also avoided it https://github.com/envoyproxy/envoy/pull/17598 but it's a bit riskier, so this seemed fine for now